src/containers/Login/page.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import Logo from 'components/Auth/Logo';
+import BackgroundLayout from 'components/BackgroundLayout';
+import backgroundImage from 'images/auth_background.png';
+
+import Form from './form';
+
+const Page = () => {
+  const logoLabel = 'Sign in to Nimble';
+  return (
+    <BackgroundLayout backgroundImage={ backgroundImage }>
+      <div className="container container-login">
```

Comment:
how about name it as `login-container` and [@extend](https://sass-lang.com/documentation/at-rules/extend) the `container` in SCSS file?

Reply:
`login-container` sounds better but is extending a bootstrap class in SCSS necessary? Cause, this means to use all other bootstrap classes like `row`, `col`, `btn` as @extend in the SCSS.

Reply:
What @namtx recommended is actually correct and recommended (see https://compass.nimblehq.co/development/code-conventions/css/#components)

![image](https://user-images.githubusercontent.com/696529/93729573-5af54380-fbef-11ea-8838-1cbdb5ded381.png)

The reason behind is that it avoids the framework naming to dictate/leak into the applications. So here the class name could be just be `container-login` or `form-login` thus avoiding 2 classes instead of one and

Reply:
Changed in [eb88898](https://github.com/rafayet-monon/surveyor/pull/13/commits/eb88898542ba6489bd75a9f7013bb68fc6a81d3e)